src/components/Dashboard/Dashboard.vue

```diff
@@ -4,35 +4,19 @@
       <v-col>
         <header class="view-header flex-column mb-0">
           <h1 class="view-header__title">FAS Staff Dashboard</h1>
-          <p class="mb-10">
+          <p class="mb-15">
             Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at
```

Comment:
We should create a ticket for all final text in FAS and assign to UI.

Reply:
Roy would be creating a ticket with all correct sentences